idsvr/provider3/saml-protocol-config.xml

```diff
@@ -0,0 +1,40 @@
+<config xmlns="http://tail-f.com/ns/config/1.0">
+    <profiles xmlns="https://curity.se/ns/conf/base">
+    <profile>
+    <id>authentication-service</id>
+    <type xmlns:auth="https://curity.se/ns/conf/profile/authentication">auth:authentication-service</type>
+      <settings>
+      <authentication-service xmlns="https://curity.se/ns/conf/profile/authentication">
+        <redirect-url-whitelist>
+          <url>https://login.example.com:8443/authn/authentication/dynamic-authenticator-oidc/</url>
```

Comment:
Should the redirect url whitelist point to the saml authenticator? Not sure if that is at all needed in this configuration.

Reply:
In order to avoid open redirects this external provider needs to pre-approve the URLs of the relying parties.

Reply:
The file is not used. I'll remove it. All the configuration is in the `config-backup.xml`